src/settings.py

```diff
@@ -211,6 +209,16 @@ def get_arg_specific_settings(args: Dict[str, Any]) -> Dict[str, Any]:
         experiment-specific args."""
 
         return dict(
+            # Maximum number of steps to run a policy when checking if it
+            # solves a task.
+            horizon=defaultdict(
```

Comment:
Maybe I'm missing something but why not do something like

```python
DEFAULT_HORIZON = 100

dict(
  horizon={
      "behavior": 1000,
      "pybullet_blocks": 1000,
  }.get(args.get("env"), DEFAULT_HORIZON)
)
```

Rather than have a default dict? Seems like the defaultdict pattern is repeated with other configs in settings as well

Reply:
i prefer this pattern because we want to keep the default value close to the overriden values (it would be bad to have DEFAULT_HORIZON at the top of file but then you have to scroll to some other place to figure out what the horizon actually is for your environment)